pkg/satellite/stream/stream.go

```diff
@@ -314,7 +317,7 @@ func (ss *satelliteStream) receiveLoop() {
 					break
 				}
 				telemetryData := telemetry.Data
-				log.Debug("received data: streamId: %v, planId: %s, framing type: %s, size: %d bytes\n", ss.streamId, planId, telemetry.Framing, len(telemetryData))
+				log.Debug("received data: streamId: %v, planId: %s, groundStationId: %s, framing type: %s, size: %d bytes\n", ss.streamId, planId, ss.groundStationId, telemetry.Framing, len(telemetryData))
```

Comment:
If ss.groundStationId is not set, this will display blank for that field.
Do we have access to this ID through the streamed data itself?

Reply:
Yep good idea, added in 552e21de75a84487d3add720190ea489ab9a1f56 👍